src/Renderer/AbstractRenderer.php

```diff
@@ -370,14 +410,14 @@ protected function checkBarcodeObject()
      * Calculate the left and top offset of the barcode in the
      * rendering support
      *
-     * @param  float $supportHeight
-     * @param  float $supportWidth
+     * @param float $supportHeight
+     * @param float $supportWidth
      * @return void
      */
     protected function adjustPosition($supportHeight, $supportWidth)
     {
         $barcodeHeight = $this->barcode->getHeight(true) * $this->moduleSize;
-        if ($barcodeHeight != $supportHeight && $this->topOffset == 0) {
+        if ($barcodeHeight !== $supportHeight && $this->topOffset === 0) {
```

Comment:
`===` and `!==` with floating point values is potentially a bit broken/risky here :thinking: